pkgs/development/python-modules/tidalapi/default.nix

```diff
@@ -17,26 +20,24 @@ buildPythonPackage rec {
     hash = "sha256-X6U34T1sM4P+JFpOfcI7CmULcGZ4SCXwP2fFHKi1cWE=";
   };
 
-  nativeBuildInputs = [
-    poetry-core
-  ];
+  nativeBuildInputs = [ poetry-core ];
 
   propagatedBuildInputs = [
     requests
     python-dateutil
+    mpegdash
+    isodate
+    ratelimit
+    typing-extensions
   ];
 
-  doCheck = false; # tests require internet access
```

Comment:
the tests in the github source all require internet access, so this line should not be removed.